.github/workflows/test.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - uses: Swatinem/rust-cache@v1
       - run: rustup target add wasm32-unknown-unknown
       - name: Test
-        run: cargo test --all --features unstable
+        run: cargo test --all --features unstable --features legacy
```

Comment:
optional: it's kinda weird that `--features` can be supplied multiple times like this. Normally I just put `--features unstable,legacy`, but up to you